source/classes.tex

```diff
@@ -6479,11 +6479,11 @@
 \end{note}
 
 \pnum
-The direct base class subobjects of \tcode{C},
+The direct base class subobjects of an object \tcode{x} of class \tcode{C},
 in the order of their declaration in the \grammarterm{base-specifier-list} of \tcode{C},
-followed by the non-static data members of \tcode{C},
```

Comment:
Why are we losing "non-static" here? As far as I'm aware, static members don't create subobjects?

Reply:
This PR is switch to use "direct member subobjects". They're not equivalent as non-static member references are not member subobjects. But given [[class.compare.default]/2](https://eel.is/c++draft/class.compare.default#2) rejects non-static member references, I think it's more appropriate to say direct member subobjects.